client.go

```diff
@@ -70,13 +70,26 @@ func (c *Client) Run(res chan *RunResults) {
 }
 
 func (c *Client) genMessages(ch chan *Message, done chan bool) {
-	for i := 0; i < c.MsgCount; i++ {
-		ch <- &Message{
-			Topic:   c.MsgTopic,
-			QoS:     c.MsgQoS,
-			Payload: make([]byte, c.MsgSize),
+	if c.MsgPayload != "" {
```

Comment:
What about refactoring that to use `switch/case` ?

Reply:
Not sure is switch/case is needed here. I only need two cases - MSGPaylod is not empty or it is empty, so if statement seems better.

Reply:
How about the following?

```go
var payload interface{}
if c.MsgPayload != "" {
	payload = c.MsgPayload
} else {
	payload = make([]byte, c.MsgSize)
}

ch <- &Message{
	Topic:   c.MsgTopic,
	QoS:     c.MsgQoS,
	Payload: payload,
}
}
```
 Sorry for a long turnaround, would you be able to update the PR?

Reply:
@krylovsk sorry for the late reply. I'll try to spin the dev environment, but I currently with another project and not sure when I'll find time to properly test this. Can you update the PR?

Reply:
Hey, just 1.5 years later I managed to get to it  :). The formatting got messy, but I'll fix that in another commit.